docs/reference.rst

```diff
@@ -152,7 +152,7 @@ Example configuration:
 
 .. _data-directory:
 
-``data-directory``
+``data-directory`` (optional, recommended)
```

Comment:
For all others, we have either the one or the other.

Reply:
For the others we have either required or optional. "recommended" is independent of that.

Reply:
You mean like

    ``type=<type>`` (recommended)

and others?

Reply:
`type=` should be (optional, recommended) as well.

Reply:
Fine for me. Same then for `bundle-formats` and `format`.